dev/VSIX/Extension/Cs/Dev17/WindowsAppSDK.Cs.Extension.Dev17.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project ToolsVersion="Current">
+<Project ToolsVersion="Current">
```

Comment:
A diff on the first line indicates a change to line endings or the BOM. Check your editor settings.